Pod/Assets/Scripts/iconVersioning.sh

```diff
@@ -43,87 +48,100 @@ shopt -u extglob
 caption="${version} ($build_num)\n${branch}\n${commit}"
 echo $caption
 
-function abspath() { pushd . > /dev/null; if [ -d "$1" ]; then cd "$1"; dirs -l +0; else cd "`dirname \"$1\"`"; cur_dir=`dirs -l +0`; if [ "$cur_dir" == "/" ]; then echo "$cur_dir`basename \"$1\"`"; else echo "$cur_dir/`basename \"$1\"`"; fi; fi; popd > /dev/null; }
+function abspath() {
+	pushd . > /dev/null
+
+	if [ -d "$1" ]; then
+		cd "$1"
+		dirs -l +0
+	else
+		cd "`dirname \"$1\"`"
+		cur_dir=`dirs -l +0`
+
+		if [ "$cur_dir" == "/" ]; then
+			echo "$cur_dir`basename \"$1\"`"
+		else
+			echo "$cur_dir/`basename \"$1\"`"
+		fi
+	fi
+
+	popd > /dev/null
+}
 
 function processIcon() {
     base_file=$1
-    
+
     cd "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
-    base_path=`find . -name ${base_file}`
-    
+    base_path=`find . -maxdepth 2 -name ${base_file}`
```

Comment:
The original or the one in my diff comment? Either way, if it's not added, having a target that has an Apple Watch app linked to it, this script throws out some errors, because the `find` command actually returns 2 files instead of one, but the code after it only expects one file. So the `-print -quit` arguments in my comment assures that the command quits after the first found file.